src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../patternfly-utilities";
+
+.pf-c-empty-state {
+  --pf-c-empty-state--MaxWidth: #{pf-size-prem(400px)}; // one off to match design
```

Comment:
would we want to set this? Shouldn't the layout deal with the width?

Reply:
from #293  @mceledonia said

> In my mockups I am using a max-width of 400px, let's try that and see how it works.

It's to address layouts such as this:

<img width="2636" alt="screen shot 2018-11-26 at 10 10 57 am" src="https://user-images.githubusercontent.com/35148959/49026520-aeb5b380-f163-11e8-9f1f-9137d1f75709.png">

Is there a case where we would want the empty state contents to be wider than 400px?

Reply:
sounds good to me :)